src/graph/graph.component.ts

```diff
@@ -626,7 +561,7 @@ export class GraphComponent extends BaseChartComponent implements OnInit, OnDest
    */
   pan(x: number, y: number): void {
     const zoomLevel = this.zoomLevel;
-    this.transformationMatrix = transform(this.transformationMatrix, translate(x / zoomLevel, y / zoomLevel));
+    this.transformationMatrix = transform(this.transformationMatrix, translate(x, y));
```

Comment:
I will revert this code change due to https://github.com/swimlane/ngx-graph/pull/98/files